frontend/src/app/features/map/DropDownButton.tsx

```diff
@@ -0,0 +1,102 @@
+import { MouseEvent, MouseEventHandler, ReactNode, useState } from 'react';
+import { Button, ButtonProps, MenuProps } from '@mui/material';
```

Comment:
I think for visual consistency across the app the menu trigger `Button` should come from `frontend/src/app/components/button/Button.tsx`, not from `@mui`

I know that organicsinfo uses Material buttons for this feature, but there it's also inconsistent with the BC gov's design system. I suggest checking with Harry about this.

Reply:
I'm ok with using Material's Menu component for now though, the style inconsistencies there are less apparent

Reply:
Will go ahead with `Button` coming from `frontend/src/app/components/button/Button.tsx`. Thanks for the feedback.